enunion/src/lib.rs

```diff
@@ -30,3 +30,17 @@ where
         .into_utf8()
         .ok()
 }
+
+#[doc(hidden)]
+pub fn merge_objects(target: &mut JsObject, sources: &[JsObject]) -> napi::Result<()> {
```

Comment:
you don't need `inline(never)` for these out line functions?

Reply:
they aren't generic, so we can just trust Rust to do the right thing (e.g. inline if only one use, etc)